src/app/content/highlights/components/cardStyles.ts

```diff
@@ -0,0 +1,205 @@
+  `}
+`;
+
+const mobileDisplay = css`
+  ${(props: CardProps) => !!props.isFocused && css`
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: unset;
+    position: fixed;
+    padding: 0;
+  `}
+  ${(props: CardProps) => !props.isFocused && css`
+    display: none;
+  `}
+`;
+
+const fadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+`;
+
+const fadeInAnimation = css`
+  animation: ${600}ms ${fadeIn} ease-out;
+`;
+
+export const mainCardStyles = css`
+  ${fadeInAnimation}
+  position: absolute;
+  padding: ${cardPadding}rem;
+  ${cardBorder}
+  ${rightSideDisplay}
+  ${disablePrint}
+
+  z-index: ${(props: CardProps) => props.zIndex};
+  transition: all 0.3s;
+
+  ${DropdownList} {
+    z-index: 1;
+  }
+
+  ${(props: {data: HighlightData}) => {
+    const data = props.data;
+
+    if (!data || !data.color) {
+      return null;
+    }
+
+    const style = highlightStyles.find((search) => search.label === props.data.color);
+
+    if (!style) {
+      return null;
+    }
+
+    return css`
+      ::before {
+        content: ' ';
+        border-radius: 0.4rem 0 0 0.4rem;
+        position: absolute;
+        top: 0;
+        left: 0
+        bottom: 0;
+        width: ${cardPadding / 2}rem;
+        background-color: ${style.focused};
+      }
+      ${theme.breakpoints.mobile(css`
+        ::before {
+          border-radius: 0.4rem 0.4rem 0 0;
+          right: 0;
+          bottom: unset;
+          width: unset;
+          height: ${cardPadding / 2}rem;
+        }
+     `)}
+    `;
+  }}
+
+  /* for some reason isTocOpen prop will not refresh after first page load and is null */
+  ${(props: CardProps) => (props.isTocOpen === null || props.isTocOpen) && css`
+    @media ${minimalWidthForCardsWithToc} {
+      /* the window is too small to show note cards next to content when the toc is open */
+      animation: none;
+      ${overlapDisplay}
+    }
+  `}
+
+  /*
+    not sure if this breakpoint is necessary, because search results have same width
+    (from the visual perspective, because values are different)
+  */
+  ${(props: CardProps) => props.hasQuery && css`
+    @media ${minimalWidthForCardsWithSearchResults} {
+      /* the window is too small to show note cards next to content when search is open */
+      animation: none;
+      ${overlapDisplay}
+    }
+  `}
+
+  @media (max-width: ${minimalWidthForCards}em) {
+    /* the window is too small to show note cards next to content even without sidebars */
+    animation: none;
+    ${overlapDisplay}
+  }
+
+  ${theme.breakpoints.mobile(css`
+    animation: none;
+    ${mobileDisplay}
+  `)}
+`;
+
+export const mainWrapperStyles = css`
+  position: relative;
+  overflow: visible;
+  z-index: ${theme.zIndex.highlightInlineCard};
+  top: 0;
+  transition: all 0.3s;
+
+  ${(props: WrapperProps) => (props.isTocOpen === null || props.isTocOpen) && css`
+    @media ${minimalWidthForCardsWithToc} {
+      top: 0 !important;
```

Comment:
why do you have important on all of these

Reply:
This `top` value is set by javascript so it has a bigger priority that css

Reply:
oh and it needs to be overridden for the inline display to work properly... interesting.... can you leave a comment in here like `override js inline styles for display of all cards while using overlap display` or something?